tests/test_ha.py

```diff
@@ -213,6 +213,10 @@ def setUp(self):
         self.ha.old_cluster = self.e.get_cluster()
         self.ha.cluster = get_cluster_initialized_without_leader()
         self.ha.load_cluster_from_dcs = Mock()
+        self.ha_dcs_orig_name = self.ha.dcs.__class__.__name__
+
+    def tearDown(self) -> None:
+        self.ha.dcs.__class__.__name__ = self.ha_dcs_orig_name
```

Comment:
Hmm.
I don't think that there is something that relies on it.
In any case we don't need to do it in setUp/tearDown, restoring right in the `test_recover_raft()` should be enough.